### DISCUSSION TITLE
Automatic Horizontal Scroll in Sidebar

### DISCUSSION THREAD

**Discussion by midwire** [atom/atom#22561] [Q&A]
Is there a way to disable the automatic horizontal scroll in the sidebar when selecting a long filename inside a folder?  

I frequently work on large projects with source code files embedded inside folders that can become nested several levels deep.

I've not seen any other discussions or references to this issue, so maybe it is just me that finds it annoying.

![atom-sidebar](https://user-images.githubusercontent.com/178386/121238622-88555080-c855-11eb-8f3c-8cc3b9ecdafa.gif)

As this .gif shows, when selecting nested filenames Atom tries to be helpful and scroll the `contents` of the sidebar to the left.

Although only a minor annoyance, it can be come tiresome to constantly have to scroll back to the left at the bottom of the sidebar in order to see files at the root level, or nested less deeply.  The ability to toggle this feature off would be very helpful in my case.

**Comment by UziTech** [comment]
What version of Atom and OS are you using? It doesn't automatically scroll for me on Atom v1.57.0 Windows 10.

**Reply by midwire** [reply]
Atom: 1.57.0
MacOS: 10.14.6 (18G8022)

I think it also happens on my Linux laptop. I'll check tonight and report back.
